app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
+  include Itms::HarvesterHelpers
+
+  class ApiError < StandardError; end
+
+  attr_reader :min_id, :downloader
+
+  def self.run(min_id: 0, downloader: HarvesterUtils::Downloader)
+    new(min_id, downloader).run
+  end
+
+  def initialize(min_id, downloader)
+    @min_id = min_id
+    @downloader = downloader
+  end
+
+  def run
+    response = downloader.get("https://opendata.itms2014.sk/uctovneDoklady?min_id=#{min_id}&limit=100")
```

Comment:
Je to nejdůležitější informace která je v podstatě konfigurací harvestera a pokud by se měnila, chtěl bych, aby byla hned na očích a nemusel jsem jí hledat uvnitř metody `run`

Řešení interpolace:

```ruby
SOURCE_DATE_URL = 'https://opendata.itms2014.sk/uctovneDoklady?min_id=%s&limit=%s'
```

V metotě run:
```ruby
response = downloader.get(SOURCE_DATA_URL % [min_id, 100])
```